python/python/tests/test_transaction.py

```diff
@@ -763,3 +763,15 @@ async def test_transaction_lastsave(
         lastsave_time = response[0]
         assert isinstance(lastsave_time, int)
         assert lastsave_time > yesterday_unix_time
+
+    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lolwut_transaction(self, redis_client: RedisClusterClient):
+        transaction = Transaction()
+        transaction.lolwut().lolwut(5).lolwut(parameters=[1, 2]).lolwut(6, [42])
+        results = await redis_client.exec(transaction)
+        assert results is not None
+
+        for element in results:
+            if isinstance(element, str):
```

Comment:
Do you get a mypy error if you remove this line? What about if you replace it with `assert isinstance(element, str)`?

Reply:
If I remove this line, there is no issue on `mypy .`, but it will fail CI/CD. That is why I added it this morning.